flyway-core/src/test/java/org/flywaydb/core/internal/dbsupport/sqlserver/SQLServerMigrationTestCase.java

```diff
@@ -327,6 +329,52 @@ public void dmlErrorsCorrectlyDetected() throws Exception {
     }
   }
 
+    @Test
+    public void msDBToolsIgnoredForEmpty() throws Exception {
```

Comment:
Testing: create the MS tools assets do ensure that they are ignored on doEmpty and doClean.